### PR TITLE
Release 2.0

### PR DESCRIPTION
**Pull request by notriddle**

### CODE REVIEW

**Comment by Eijebong** [comment]
Is there nothing else we want to do before ?

**Comment by notriddle** [comment]
There's no other [2.0 wishlist](https://github.com/rust-ammonia/ammonia/issues?q=label%3A%222.0+wishlist%22) issues, which is specifically a list of breaking changes that we wanted to make (which was the whole point of releasing 2.0).

If there's something else *you'd* like to do before releasing 2.0, please say what it is!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 *
 
+# 2.0.0
+
+* Add "script" and "style" to the default set of [clean content tags]
+* Make all iterator-accepting APIs use `IntoIterator` and `Borrow`, so that you can pass slices directly to them.
+
+[clean content tags]: https://docs.rs/ammonia/1.2.0/ammonia/struct.Builder.html#method.clean_content_tags
```
Comment (Eijebong) [inline CHANGELOG.md]: You might want to link to 2.0.0 directly ?

**Comment by Eijebong** [comment]
I don't have anything in mind, just asking :p

File: CHANGELOG.md (hunk 1 above)
Comment (notriddle) [reply CHANGELOG.md]: Yeah...

**Comment by notriddle** [comment]
Sorry. Found one other bug in the changelog. Hit it again.

**Comment by notriddle** [comment]
bors r= @Eijebong

**Pull request merged**